autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -65,6 +70,8 @@ public class ScalingRequest implements Runnable {
 
     private final ScalingPolicy policy;
 
+    private final ScalingTrigger scalingTrigger;
```

Comment:
What happens if there's a failure communicating with Cloudbreak itself? We need to be capturing this as a scaling failure as well.